frida_mode/ts/lib/afl.ts

```diff
@@ -266,11 +266,12 @@ class Afl {
    * See `AFL_FRIDA_SECCOMP_FILE`. This function takes a single `string` as
    * an argument.
    */
+  /* dk how to exlcude
```

Comment:
Should be able to re-instate this if the `__APPLE__` model for excluding seccomp is usable.